patches/chrome-browser-content_settings-tab_specific_content_settings.cc.patch

```diff
@@ -1,20 +1,20 @@
 diff --git a/chrome/browser/content_settings/tab_specific_content_settings.cc b/chrome/browser/content_settings/tab_specific_content_settings.cc
-index d2e6db2a8fd1d7aec6f31f25ed102150fc0780f5..883f19ff4c296d3d06ea4443e94573c17c4cb919 100644
+index a736dbf5c2ee2cdc98173b14d9405afb067e4577..0413cbeba3453c1c90fd467c3fab3ea6518bd541 100644
 --- a/chrome/browser/content_settings/tab_specific_content_settings.cc
 +++ b/chrome/browser/content_settings/tab_specific_content_settings.cc
-@@ -239,6 +239,7 @@ bool TabSpecificContentSettings::IsContentBlocked(
-       content_type == CONTENT_SETTINGS_TYPE_ADS ||
-       content_type == CONTENT_SETTINGS_TYPE_SOUND ||
-       content_type == CONTENT_SETTINGS_TYPE_CLIPBOARD_READ ||
-+      content_type == CONTENT_SETTINGS_TYPE_AUTOPLAY ||
-       content_type == CONTENT_SETTINGS_TYPE_SENSORS) {
+@@ -240,6 +240,7 @@ bool TabSpecificContentSettings::IsContentBlocked(
+       content_type == ContentSettingsType::ADS ||
+       content_type == ContentSettingsType::SOUND ||
+       content_type == ContentSettingsType::CLIPBOARD_READ ||
++      content_type == ContentSettingsType::AUTOPLAY ||
```

Comment:
fyi for next time we should convert to a `define` if we have to modify the patch anyway

Reply:
can be done in a follow up